code/provision/playbooks/roles/kubernetes/node/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Recreate kube-dns
```

Comment:
I think the recommendation is to use CoreDNS rather than KubeDNS

Reply:
Good shout. Probably not one for this PR but worth considering.